cli/factory.rs

```diff
@@ -563,6 +566,9 @@ impl CliFactory {
   pub fn feature_checker(&self) -> &Arc<FeatureChecker> {
     self.services.feature_checker.get_or_init(|| {
       let mut checker = FeatureChecker::default();
+      if self.options.unstable_bare_node_builtlins() {
+        checker.enable_feature("bare-node-builtins");
+      }
```

Comment:
I'm not sure it makes sense to pass this to the feature checker unless the feature is being used with the runtime code. For example, in this case we could just call `self.options.unstable_bare_node_builtins()` above on line 380.

Reply:
Maybe this is just an overhead. Updated